Solutions/Forescout Host Property Monitor/Playbooks/DnsniffEventPlaybook-1.json

```diff
@@ -0,0 +1,386 @@
+{
+    "$schema":  "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion":  "1.0.0.0",
+    "parameters":  {
+        "PlaybookName":  {
+            "defaultValue":  "DnsniffEventPlaybook-1",
```

Comment:
Can we give a suitable name for the playbook file and playbook. This would help the users understand the operation of the playbook better.

Reply:
Our customers are familiar with and understand the name DnsniffEvent. And we are planning to add more examples later, so we added a number to the name.